Runtime/CesiumGlobeAnchor.cs

```diff
@@ -415,78 +415,99 @@ public void SetPositionUnity(double x, double y, double z)
         /// </para>
         /// <list type="bullet">
         /// <item>
-        /// If none of the position properties on this instance have been set, it updates them all
+        /// If the <see cref="localToGlobeFixedMatrix"/> transform has not yet been set, it is computed
         /// from the game object's current `Transform`.
         /// </item>
         /// <item>
-        /// If the game object's `Transform` has changed since the last time `Start`, this method,
+        /// If the game object's `Transform` has changed since the last time `OnEnable`, this method,
```

Comment:
```suggestion
        /// If the game object's `Transform` has changed since the last time `OnEnable`, or if this method
```

Reply:
Actually I think I misunderstood what this comment was saying, so it really should say:
```suggestion
        /// If the game object's `Transform` has changed since the last time `OnEnable`, or if a position 
        setter was called, this method ...
```